builtin/game/item_entity.lua

```diff
@@ -36,6 +36,9 @@ core.register_entity(":__builtin:item", {
 	itemstring = '',
 	physical_state = true,
 	age = 0,
+	-- The items are mergeable as long as this property is not false,
+	-- so we use nil to save a few bytes unless it will be set.
+	mergeable = nil,
```

Comment:
@RobertZenz: I'd use a name like `nonmergeable` (although not necessarily that particular name).